docs/manifest.json

```diff
@@ -509,6 +503,12 @@
 		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/packages/components.md",
 		"parent": null
 	},
+	{
+		"title": "AccessibleSvg",
```

Comment:
To clarify, this is auto-generated title? As code is concerned, this would be incorrect capitalization (vs. `AccessibleSVG`, which as best I can tell is correctly used elsewhere).

Reply:
https://github.com/WordPress/gutenberg/blob/master/docs/tool/manifest.js#L57 - yes it is machine-generated. We could try to read it from the title of `README.md` as a first thing before using what we have at the moment.